demos/python_demos/human_pose_estimation_3d_demo/README.md

```diff
@@ -0,0 +1,88 @@
+# 3d Human Pose Estimation Python* Demo
+
+This demo demonstrates how to run 3d Human Pose Estimation models using OpenVINO&trade;.
+
+> **NOTE**: Only batch size of 1 is supported.
+## How It Works
+
+The demo application expects a 3d human pose estimation model in the Intermediate Representation (IR) format.
```

Comment:
Other demos do not mention it. Also there is `models.lst` for this.

Reply:
> Other demos do not mention it.

[Yes](https://github.com/opencv/open_model_zoo/blob/5c00f9656e10091cf5c9d3c38877d39c17abc826/demos/crossroad_camera_demo/README.md) [they](https://github.com/opencv/open_model_zoo/blob/5c00f9656e10091cf5c9d3c38877d39c17abc826/demos/gaze_estimation_demo/README.md) [do](https://github.com/opencv/open_model_zoo/tree/5c00f9656e10091cf5c9d3c38877d39c17abc826/demos/human_pose_estimation_demo).

> Also there is models.lst for this.

That's a fair point, but you haven't referenced that either.

Reply:
Actually no. I thought models from `models.lst` are implicitly implied. Should the notice about `models.lst` be added?

Reply:
Ideally we should just reference `models.lst`; however, at this point it won't work well, since a user browsing the online copy of the documentation won't be able to open the `models.lst` file. So you should just name the model directly.